google-account-plugin/src/com/google/gct/login/GoogleLogin.java

```diff
@@ -565,8 +566,23 @@ public String getInfo() {
    * An implementation of {@link UiFacade} using Swing dialogs and external browsers.
    */
   private class AndroidUiFacade implements UiFacade {
+
+    /**
+     * number of characters to wrap lines after in the logout message
+     */
+    public static final int WRAP_LENGTH = 60;
     private GoogleLoginActionButton myButton;
     private volatile CancellableServerReceiver receiver = null;
+    private GoogleLoginMessageExtender[] messageExtenders;
+
+    public AndroidUiFacade() {
+      GoogleLoginMessageExtender[] messageExtenders = Extensions.getExtensions(GoogleLoginMessageExtender.EP_NAME);
+      if (messageExtenders.length == 0) {
+        this.messageExtenders = new GoogleLoginMessageExtender[0];
```

Comment:
this.messageExtenders is null by the field declaration, this saves a null check whenever we want to iterate over the array. I can move the initialization to the field declaration though.